web/beacon-app/src/features/auth/components/Login/LoginForm.tsx

```diff
@@ -59,6 +63,7 @@ function LoginForm({ onSubmit, isDisabled, isLoading, initialValues }: LoginForm
             <Trans>Log in</Trans>
           </Button>
         </div>
+        <div id="google-recaptcha" className="flex flex-col"></div>
```

Comment:
I moved the recaptcha div so that the `Forgot Password` link and the `Log In` button would be evenly spaced out. I didn't want to remove the div if it is necessary.